### DISCUSSION TITLE
document how to get back the "show source" footer link

### DISCUSSION THREAD

**Discussion by LuisBL** [pradyunsg/furo#516] [Potential Issue] [answered]
In the discussion https://github.com/pradyunsg/furo/discussions/374

Is there any way to bring back the show source footer link, it is very useful to help new comers to rst to understand what rst code was used to render a page they are looking at.

Just the link to `_build/html/_sources/src/page_we_are_in.rst.txt`

**Comment by pradyunsg** [comment, marked as answer]
2ce7e89cb799bcb1312b711b17fca90d2979fa21 brought this link back. See https://pradyunsg.me/furo/customisation/top-of-page-buttons/#linking-to-copied-sources for the relevant documentation.
